server/supernova/sc/sc_plugin_interface.cpp

```diff
@@ -563,6 +563,16 @@ int do_asynchronous_command(
     return 0;
 }
 
+bool send_message_from_RT(World* world, struct FifoMsg& msg) {
```

Comment:
is it OK to use a reference here for C ABI compatibility? or should it be a pointer? (naive question)

Reply:
the plugin API itself defines it like that. in practice, references are implemented as pointers (although the C++ standard doesn't mandate this), so this is safe.